sentinel-dashboard/src/test/java/com/alibaba/csp/sentinel/dashboard/repository/metric/InMemoryMetricsRepositoryTest.java

```diff
@@ -95,47 +127,41 @@ public void testExpireMetric() {
         inMemoryMetricsRepository.save(entry);
 
         List<MetricEntity> list = inMemoryMetricsRepository.queryByAppAndResourceBetween(
-            DEFAULT_EXPIRE_APP, DEFAULT_RESOURCE, now - 2 * EXPIRE_TIME, now + EXPIRE_TIME);
+                "testExpire", "testResource", now - EXPIRE_TIME, now);
```

Comment:
Why change the range here？I think previous `2 * EXPIRE_TIME` is better, since it proves that the `expireEntry` has not exist in repository now.

Reply:
> Why change the range here？I think previous `2 * EXPIRE_TIME` is better, since it proves that the `expireEntry` has not exist in repository now.

In this case what want to be proved is  entry which  exceed `EXPIRE_TIME` will be eliminate not the `2 * EXPIRE_TIME`.  `2 * EXPIRE_TIME` can pass the test case ， but not strictly.

Reply:
The last assertion has proved the case, and the new range here is more clearer semantically and strictly. I agree, it's ok.